tasks/main.yml

```diff
@@ -7,6 +7,8 @@
     # TODO: run Ansible deployment on the host and wait for it to complete
   when: openshift_is_ansible_host
 
+- import_tasks: node.yml
```

Comment:
Why deploy the SSH keys on Ansible hosts?

Reply:
See my comment on the PR. Additional Ansible hosts can be regular OpenShift nodes as well, we don't forbid that, so the other Ansible hosts need access to them. Otherwise, if you had two Ansible hosts, you would need two Ansible runs, one on host 1, to deploy host 1 and the rest of the cluster, and an additional run on host 2, to deploy host 2 (and the rest of the cluster again).